package/MDAnalysis/coordinates/PDB.py

```diff
@@ -234,7 +234,10 @@ class PDBReader(base.ReaderBase):
 
     *The columns 73-76 are not part of the official PDB format but are used by
     some programs to store/operate the segment ID. For instance, Chimera_ assigns
-    it as the attribute `pdbSegment` to allow command-line specification.
+    it as the attribute `pdbSegment` to allow command-line specification. If you
+    would like to force the use of chainID as the segID when reading PDB, please
+    use the kwarg `force_chainids_to_segids=True`. This will prioritize the
```

Comment:
On re-reading I realized that this is out of place because it implies that the PDBReader has the ``force_chainids_to_segids`` kwarg, which it doesn't. You have to rewrite it to make it more clear that you are talking about the PDBParser.

I suggest you put the sentence into a SeeAlso box and explicitly link to PDBParser.